### PR TITLE
when sailing into a null region SailThroughCheck should always return

### PR DESCRIPTION
**Pull request by sharcashmo**
SailThroghCheck should always return false when sailing into null region

Included some changes in coding style. I don't think it is standarized, but it seems as latest changes always include the brackets {} in if clauses even if they're a single line block

### CODE REVIEW

File: object.cpp [hunk 1]
```diff
@@ -609,27 +609,41 @@ int Object::FleetLoad()
 int Object::SailThroughCheck(int dir)
 {
 	if (IsFleet()) {
+		// if target region doesn't exist, cannot be sailed into
+		if (!region->neighbors[dir]) {
```
Comment (artyomtrityak) [inline object.cpp:613]: more common style in code is not use brackets for single line if like

`if (!region->neighbors[dir]) return 0;`

I am not a big fan of it but to keep code consistent...

**Comment by sharcashmo** [comment]
I've seem both in the code, and I actually thought that was the contrary.

Maybe we should take a decision on that. Not to change all existing code,
but at least for the new one or the fix we do. But as you prefer, I'll
stick to what you decide in this matter :)

**Comment by artyomtrityak** [comment]
i would like to keep it consistent with single like statements

**Comment by sharcashmo** [comment]
It is ok for me, as you prefer.

Maybe I saw the wrong part of the code :D, i got the impression than older
code was without brackets, while newer one was with them :)

But both are ok for me.

Should I change the source code to remove brackets in single line if
statements?

**Comment by artyomtrityak** [comment]
@sharcashmo to be honest i do not care much - both look good! Let's merge it.

**Review by artyomtrityak — APPROVED** [review]

**Pull request merged**